### PR TITLE
PIC-3759: Stop filters persisting

### PR DESCRIPTION
**Pull request by ArronChave3**

### CODE REVIEW

File: tests/routes/view.test.js [hunk 1]
```diff
@@ -748,6 +748,7 @@ describe('Routes', () => {
       return request(app)
         .get('/B14LO/outcomes?hearingDate=ASC&outcomeType=ADJOURNED')
         .then(response => {
+          console.log(response)
```
Comment (johnyfarrar) [inline tests/routes/view.test.js:751]: Console Log

File: server/views/case-list.njk [hunk 2]
```diff
@@ -309,6 +311,9 @@
         {% endif %}
     {% endif -%}
 
+    hello
```
Comment (andrewcharnley) [inline server/views/case-list.njk:314]: Hello Arron!

**Review by andrewcharnley — CHANGES_REQUESTED** [review]
L314

File: server/views/case-list.njk (hunk 2 above)
Comment (ArronChave3) [reply server/views/case-list.njk:314]: hi

**Review by andrewcharnley — APPROVED** [review]

**Pull request merged**